state/rds-aurora.yaml

```diff
@@ -102,10 +104,21 @@ Parameters:
     Type: 'String'
     Default: master
   DBMasterUserPassword:
-    Description: 'The master password for the DB instance (ignored when DBSnapshotIdentifier is set, value used from snapshot).'
+    Description: 'The master password for the DB instance (ignored when DBSnapshotIdentifier is set, value used from snapshot. Also ignored when UseSecretsManager is set to true).'
     Type: String
     NoEcho: true
     Default: ''
+  UseSecretsManager:
```

Comment:
I would prefer to enable the secret in situations where `DBMasterUserPassword` is empty but no `DBSnapshotIdentifier` is specified?

Reply:
sounds good, will update